### DISCUSSION TITLE
GameCube 3V3 to power ESP32?

### DISCUSSION THREAD

**Discussion by yo1dog** [darthcloud/BlueRetro#460] [Q&A]
Has anyone had luck powering the ESP32 (without devkit) from GameCube's 3V3 rail?

Internal install would be much simpler if LDO on 5v is not required.

**Comment by darthcloud** [comment]
I think @nostalgic-indulgences https://github.com/nostalgic-indulgences/BlueRetro_Internal_Installation does that.
